src/NLog/LogFactory.cs

```diff
@@ -90,14 +90,53 @@ public class LogFactory : IDisposable
         /// </remarks>
         public event EventHandler<LoggingConfigurationChangedEventArgs> ConfigurationChanged;
 
-        private static event EventHandler<EventArgs> LoggerShutdown;
+        private static event EventHandler<EventArgs> _loggerShutdown;
 
         /// <summary>
-        /// Initializes static members of the LogManager class.
+        /// Event that is raised when the logging system is shutting down.
         /// </summary>
-        static LogFactory()
+        public static event EventHandler<EventArgs> LoggerShutdown
```

Comment:
Can it become `internal` instead of `public`, since the public API should remain unchanged.